docs/ballerina-by-example/examples/grpc-unary-non-blocking-stub/client/grpc-unary-non-blocking-stub.pb.bal

```diff
@@ -0,0 +1,42 @@
+package client;
```

Comment:
Please add license headers to these bal source files. Please fix for other cases.